tasks/fh-build.js

```diff
@@ -100,26 +101,10 @@ module.exports = function(grunt) {
     }
   });
 
+  var lintTargets = ['*.js', 'lib/**/*.js', 'bin/**/*.js'];
   grunt.config.merge({
-    jshint: {
-      options: {
-        jshintrc: '.jshintrc'
-      },
-
-      fh: ['*.js', 'lib/**/*.js', 'bin/**/*.js']
-    }
-  });
-
-  grunt.config.merge({
-    plato: {
-      fh: {
-        options : {
-          jshint : grunt.file.readJSON('.jshintrc')
-        },
-        files: {
-          'plato': ['lib/**/*.js']
-        }
-      }
+    eslint: {
+      target: lintTargets
```

Comment:
Do you think there would be benefit to having this 'lintTargets' be something that can be overridden at a per-project basis? I imagine it's possible, but I haven't looked into it.